API authentication.md

```diff
@@ -0,0 +1,21 @@
+# API authentication 
+
+## Overview 
+
+Verfies the identity of the user. To make transactions on HotWax’s platform secure, all apps connecting with HotWax API might need to authenticate when making API requests.
+
+## Authentication Method Used
+
+### Bearer token authentication
```

Comment:
we have a video about generating and retrieving bearer tokens, maybe we can use that here?

Reply:
okay I am adding it here